frontend/src/app/organization/organization.service.ts

```diff
@@ -48,7 +50,32 @@ export class OrganizationService {
         err => this.contactStatusEvent.next({ data: contactData, status: 'error' }),
       );
   }
-
+  createOrganization(newOrganization: Organization) {
+    this.http.post(this.url, newOrganization, { observe: 'response' })
+    .subscribe(
```

Comment:
I guess we don't need to observe the response and check the status here.
All statuses from 200 to 299 are handled by success function (first one in the supscription) and it means that the request was resolved correctly by backend.
Other statuses are handled in error function (the second one).

```
this.http.post(this.url, newOrganization)
.subscribe(
  (data: ??? // I guess this will be of type organization) =>  {   
        this.createOrganizationEvent.next({ data, status: 'success' });
    },
    error => this.createOrganizationEvent.next({ data:error, status: 'error' })
 );
```

Also, we send only "success" or "error" status to the event, so we could instead use a boolean. So instead of sending `{ data, status: 'success' }` we can send `{ data, success: true}` and instead of `{ data: error, status: 'error' }` just `{ data, success: false}` .